src/app/components/elements/Dashboard.tsx

```diff
@@ -69,15 +77,28 @@ export const Dashboard = ({
   );
 
   const loggedInContent = (
-    <Row className="pt-4">
-      <Col md="12" lg="5" className={"pt-lg-4"}>
-        <Container className={"mb-4"}>
-          <h1 id="homepageName">Welcome {user?.loggedIn && user.givenName}</h1>
-        </Container>
-        <ShowMeButtons className={"pt-xl-2"} />
-        {/*<img id="homepageHeroImg" className="img-fluid" alt="Three Computer Science students studying with two laptops, one with code on the screen" src="/assets/ics_hero.svg" />*/}
-      </Col>
-      <PromoOrFeaturedNews contentType={user?.loggedIn && user.role === "TEACHER" && promoItem ? "promo" : "news"} />
+    <Row className="py-4">
+      {!expanded && (
+        <Col md="12" lg="5" className={"pt-lg-4"}>
+          <Container className={"mb-4"}>
+            <h1 id="homepageName">Welcome {user?.loggedIn && user.givenName}</h1>
+          </Container>
+          <ShowMeButtons className={"pt-xl-2"} />
+        </Col>
+      )}
+      {isTeacherOrAbove(user) ? (
+        <PromoOrFeaturedNews contentType={promoItem ? "promo" : "news"} />
+      ) : (
+        <ShowLoading
+          until={questionData}
+          thenRender={(questionData) => (
+            <Col md="12" lg={expanded ? "12" : "7"}>
+              {questionData && <QuestionCard setExpanded={setExpanded} questionData={questionData} />}
```

Comment:
Is `questionData &&` actually needed? I would have thought that if you reach this point it means that `questionData` won't be falsy.

Reply:
In that case, the following should do the trick:
```typescript
thenRender={(questionData: IsaacQuestionPageDTO[]) => (
```